### DISCUSSION TITLE
Tdarr error

### DISCUSSION THREAD

**Discussion by travwide** [community-scripts/ProxmoxVE#293] [Q&A] [answered]
I'm getting the following error when trying to deply tdarr. 
 \/dev/dri/renderD128 is not a device
 |
[ERROR] in line 657: exit code 0: while executing command pct start "$CTID"

WHen trying to start the LXC in Proxmox I receive this error.
TASK ERROR: /dev/dri/renderD128 is not a device

```
  ______    __               
 /_  __/___/ /___  __________
  / / / __  / __ `/ ___/ ___/
 / / / /_/ / /_/ / /  / /    
/_/  \__,_/\__,_/_/  /_/     
                             
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 114
Using Hostname: tdarr
Using Disk Size: 4GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Tdarr LXC using the above default settings
 ✓ Using containers for Template Storage.
 ✓ Using containers for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 114 was successfully created.
 \/dev/dri/renderD128 is not a device
 |
[ERROR] in line 657: exit code 0: while executing command pct start "$CTID"
```

**Comment by MickLesk** [comment]
Thats Not a Script issue, thats an issue with your Proxmox

**Comment by havardthom** [comment, marked as answer]
Tdarr sets up hardware acceleration, perhaps a permission issue with `/dev/dri/renderD128`. What is the output of `ls -l /dev/dri` on your host?

**Reply by travwide** [reply]
root@pve:~# ls -l /dev/dri
total 0
drwxr-xr-x 2 root root      60 Nov 19 17:27 by-path
crw-rw---- 1 root video 226, 0 Nov 19 17:27 card0
---------- 1 root root       0 Nov 19 17:33 renderD128

**Comment by travwide** [comment]
Looks like my pemission were off. I had to do the following,.
```
rm /dev/dri/renderD128
mknod /dev/dri/renderD128 c 226 128
chown root:video /dev/dri/renderD128
chmod 660 /dev/dri/renderD128
```
Verify the correct permissions and rerun the script.

root@pve:~# ls -l /dev/dri
total 0
drwxr-xr-x 2 root root        60 Nov 19 17:27 by-path
crw-rw---- 1 root video 226,   0 Nov 19 17:27 card0
crw-rw---- 1 root video 226, 128 Nov 20 17:04 renderD128
